extensions/cache/runtime/src/main/java/io/quarkus/cache/CaffeineCache.java

```diff
@@ -11,4 +14,19 @@ public interface CaffeineCache extends Cache {
      * @return a set view of the keys contained in this cache
      */
     Set<Object> keySet();
+
+    /**
+     * Returns the future associated with {@code key} in this cache, or {@code null} if there is no
+     * cached future for {@code key}. This method is delegating to the
+     * {@link com.github.benmanes.caffeine.cache.AsyncCache#getIfPresent(Object)}, while recording the cache stats if they are
+     * enabled.
+     *
+     * @param key key whose associated value is to be returned
+     * @return the future value to which the specified key is mapped, or {@code null} if this cache
+     *         does not contain a mapping for the key
+     * @throws NullPointerException if the specified key is null
+     * @see com.github.benmanes.caffeine.cache.AsyncCache#getIfPresent(Object)
+     */
+    @Nullable
```

Comment:
This annotation makes sense, but it's not used elsewhere in the Quarkus codebase AFAICS. @gsmet Is it ok to keep it here?

Reply:
I can remove it if needed? let me know, so we can finalize this :)

Reply:
Let's remove it and merge this PR :smiley:

Reply:
Removed, resolving.